### DISCUSSION TITLE
Where can i find more icon for the msfs2020 streamdeck?

### DISCUSSION THREAD

**Discussion by rug4ever** [nguyenquyhy/Flight-Tracker-StreamDeck#41] [General]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by Virtual-Chris** [comment]
I posted a bunch of buttons that I've scraped together from various places as well as ones I've created here...
https://drive.google.com/file/d/1k9XiFj_eoton9Acemik07sOL_uzel7qL/view?usp=sharing

**Comment by nelscougar** [comment]
So cute !!! Good job !! :) 

----- Mail original -----

De: "Virtual-Chris" <notifications@github.com> 
À: "nguyenquyhy/Flight-Tracker-StreamDeck" <Flight-Tracker-StreamDeck@noreply.github.com> 
Cc: "Subscribed" <subscribed@noreply.github.com> 
Envoyé: Jeudi 29 Octobre 2020 00:37:30 
Objet: Re: [nguyenquyhy/Flight-Tracker-StreamDeck] Where can i find more icon for the msfs2020 streamdeck? (#41) 



I posted a bunch of buttons that I've scraped together from various places as well as ones I've created here... 
https://drive.google.com/file/d/1k9XiFj_eoton9Acemik07sOL_uzel7qL/view?usp=sharing

**Comment by vicentezc** [comment]
Hey!

In case you are interested, I created some custom icons for the A320 Neo: https://drive.google.com/file/d/14vMiS0D4oPujUWVJdTccZm9y27Shgl_Z/view?usp=sharing

Many of them cannot be programmed at the moment in MSFS though (specially the ones in EFIS and ECAM folders), but I already created the icons in preparation for the future.

Some images were taken from this link: http://a320dp.com/A320_DP/panels.html
Note that the light switches may be inverted, but I chose to have the on position above and the off position below.

Cheers!

Vicente

**Comment by jplewis01** [comment]
I'm Building a an Icon Template File to create build blocks in image layers in Paint.NET (simple inexpensive and effective) Version 2 should be out today and I will post a link to in in GitHub  .

**Comment by rug4ever** [comment]
pls more Info and where can i find the file?

**Comment by rmroc451** [comment]
@rug4ever if you click his name and go check his repos, they are listed [here](https://github.com/jplewis01/Streamdeck_FSPanels-).

**Comment by rug4ever** [comment]
thx

but not my fav Icons

**Comment by rmroc451** [comment]
transferring to discussion area.
